server/app-engine/static/vehicle-assignments.js

```diff
@@ -340,22 +342,22 @@ function handleKey(id) {
         util.log('- fromDate: ' + fromDate);
         util.log('- toDate  : ' + toDate);
 
-        for (var item of items) {
+        for (item of items) {
             if (item.type === 'block' && item.status === 'assigned') {
-                var id = item.label;
-                util.log('-- id: ' + id);
+                var itemID = item.label;
+                util.log('-- itemID: ' + itemID);
                 var vid = item.vehicle;
                 util.log('-- vid: ' + vid);
-                var block = blockMap[id];
+                var block = blockMap[itemID];
```

Comment:
Shouldn't we use `let` everywhere instead of `var` for local variables?

Reply:
yeah - I was trying to keep scope of this change down but...done!